announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm

```diff
@@ -403,6 +403,11 @@ function resizeFrame(){
 
 			#if ($notification)
 				<hr class="itemSeparator" />
+
+				#if (! $site.isPublished())
```

Comment:
```suggestion
				#if (!$site.isPublished())
```

Reply:
Sounds good. I'll force push into this branch this suggestion.